.github/workflows/pytest-min-versions.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
     - name: install min_requirements.txt
       run: |
         pip install -r min_requirements.txt
+        pip install h5py==3.1.0 netCDF4==1.5.6 numpy==1.20.1
```

Comment:
The minimum versions are constrained in setup.cfg so if one wants to update them that should be changed there. Hopefully we do not need to bump the minimal supported versions to the very latest versions?